teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +128,460 @@ requirements, and provenance attestations can be made to meet software supply ch
+   and a working example provided.
+7. The resource MAY follow and demonstrate best practices, such as the
+   [`Task` authoring recommendations][task-authoring-recommendations].
+
+#### Tekton Official Catalog
+
+To provide a corpus of high quality resources that users can rely on, 
+we propose creating an Official Catalog with high maintenance, testing 
+and security standards. 
+
+The Official Catalog will be in a new repository named 
+`tektoncd/catalog-official`. 
+
+##### Requirements
+
+These are requirement that a resource must meet to be added to the 
+Official Catalog:
+
+1. The resource MUST comply with the contract defined in
+   [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+2. The resource MUST not define an OWNER file, given that it is owned
+   and maintained by Tekton Catalog Maintainers who are defined at the
+   root of the Catalog. The ownership responsibilities are described
+   [below](#responsibilities).
+3. The resource MUST have automated testing using Tekton for dogfooding.
+   If there are any failures, they MUST be resolved as soon as possible;
+   the SLO is one week. The automated testing is discussed further
+   [below](#automated-testing-and-dogfooding-1).
+4. The resource MUST be scanned for common vulnerabilities and exposures.
+   If any issues are discovered, they MUST be patched or disclosed as soon as
+   possible; the SLO is one week. The scanning for CVEs is discussed further
+   [below](#image-scanning-for-common-vulnerabilities-and-exposures-cves-1).
+5. The resource MUST support verification as proposed in
+   [TEP-0091: Verified Remote Resources][tep-0091]. For now, it MUST be
+   published to a public OCI registry as a [Tekton Bundle][bundle] and
+   signed by Tekton. In the future, it MUST be updated to support accepted
+   proposal of TEP-0091 depending on the direction of
+   [TEP-0060: Remote Resource Resolution][tep-0060].
+   The verification is discussed further [below](#verified-remote-resources-1).
+6. The resource MUST be well documented with all configuration options described
+   and a working example provided.
+7. The resource MUST follow and demonstrate best practices, such as the
+   [`Task` authoring recommendations][task-authoring-recommendations].
+
+#### Promotion from Community Catalog to Official Catalog
+
+Promoting a resource from the Community Catalog to the Official Catalog 
+should be an exception, not the norm. The set of resources in the 
+Official Catalog should relatively small so that it's sustainable for the 
+Tekton Catalog Owners to maintain. 
+
+A resource in the Community Catalog may get promoted to the Official Catalog 
+if:
+* _Quality_: It meets the [requirements of the Official Catalog](#requirements-1).
+* _Bandwidth_: The [Tekton Catalog Maintainers][catalog-owners] approve its
+  promotion based on its usefulness to the community and existing bandwidth
+  to maintain the resource.
+
+If a resource is promoted to the Official Catalog:
```

Comment:
what about images building? should the official repo use "upstream" image or the official catalog rebuild the images, the advantages is that we are not dependent of an external image that may or may not be vulnerable.